README.md

```diff
@@ -39,6 +39,8 @@ On Linux, you may install the .deb file, then run `/opt/rcv/bin/RCTab` to launch
     
     If the expected version isn't returned, you'll need to follow the instructions [here](https://www.java.com/en/download/help/path.xml) on how to set your Java path.
 
+    If you are using Linux or MacOS and need to regularly switch between Java versions, consider installing [jEnv](https://www.jenv.be/). For a list of the Java versions installed on your machine, run `/usr/libexec/java_home -V` on MacOS or `update-alternatives --config java` on Linux.
```

Comment:
> `/usr/libexec/java_home -V`

Does that mean versions of Java registered with `jenv`?

Because on my Mac, not all instances of Java showed up there.
```
$ /usr/libexec/java_home -V 
Matching Java Virtual Machines (1):
    17.0.6 (arm64) "Oracle Corporation" - "Java SE 17.0.6" /Library/Java/JavaVirtualMachines/jdk-17.jdk/Contents/Home
/Library/Java/JavaVirtualMachines/jdk-17.jdk/Contents/Home

$ ls ~/Library/Java/JVMs 
temurin-jdk-20.0.1+9

$ ls /Users/my-other-account/Library/Java/JavaVirtualMachines
.corretto-20.0.2.1.intellij	corretto-20.0.2.1
```